core/Symbols.cc

```diff
@@ -1048,7 +1048,10 @@ string_view SymbolRef::showKind(const GlobalState &gs) const {
 }
 
 string_view ClassOrModuleRef::showKind(const GlobalState &gs) const {
-    if (dataAllowingNone(gs)->isClassOrModuleClass()) {
+    auto sym = dataAllowingNone(gs);
+    if (!sym->isClassModuleSet()) {
+        return "class-or-module"sv;
```

Comment:
open to suggestions of better names here

Reply:
Another option is just returning `"module"` here as well. I don't have strong preference.